refm/api/src/_builtin/Struct

```diff
@@ -220,10 +220,10 @@ p Foo.new.members  # => [:foo, :bar]
 --- to_a -> [object]
 構造体のメンバの値を配列にいれて返します。
 
-例えば以下のようにして passwd のエントリを出力できます。
-
-  require 'etc'
-  print Etc.getpwuid.values.join(":"), "\n"
+例:
```

Comment:
Can you replace the indented sample code with `#@samplecode`?

see also https://github.com/rurema/doctree/issues/1546

Reply:
Thanks for your help, I replace to `@samplecode` instead of indented code.